pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1592,3 +1600,10 @@ func (c *VolumeBindingChecker) predicate(pod *v1.Pod, meta algorithm.PredicateMe
 	glog.V(5).Infof("All PVCs found matches for pod %v/%v, node %q", pod.Namespace, pod.Name, node.Name)
 	return true, nil, nil
 }
+
+// RegisterPodFitsResourcesPredicateMetadataProducer registers a PredicateMetadataProducer used by PodFitsResources
+func RegisterPodFitsResourcesPredicateMetadataProducer(scalaResourcesHandleByExtenders sets.String) {
```

Comment:
Not sure if we really need this function. Please consider calling `RegisterPredicateMetadataProducer` directly.

Reply:
`predicateMetadata` is an un-exported type and I can't calling RegisterPredicateMetadataProducer directly. Should I make it exported ?

Reply:
Oh, I see. This is fine. We should probable move this function to metadata.go.